### PR TITLE
Feature/cache endpoint

### PR DESCRIPTION
**Pull request by izzyblues**

### CODE REVIEW

File: author_handlers_test.go [hunk 1]
```diff
@@ -65,6 +69,25 @@ func TestShouldReturn200AndAuthorsCount(t *testing.T) {
 	assert.Equal(t, "2", actualOutput, "Response body should contain the count of available authors")
 }
 
+func TestShouldReturn500WhenAuthorsCountIsCalledAndCacheRefreshFails(t *testing.T) {
+	mbs := new(MockedBerthaService)
+	mbs.On("getAuthorsCount").Return(2)
+	mbs.On("refreshCache").Return(errors.New("I hate Luca!"))
```
Comment (khatton-ft) [inline author_handlers_test.go]: :cry:

File: author_handlers_test.go [hunk 2]
```diff
@@ -65,6 +69,25 @@ func TestShouldReturn200AndAuthorsCount(t *testing.T) {
...
+	startCuratedAuthorsTransformer(mbs)
+	defer curatedAuthorsTransformer.Close()
+
+	resp, err := http.Get(curatedAuthorsTransformer.URL + "/transformers/authors/__count")
+	if err != nil {
```
Comment (khatton-ft) [inline author_handlers_test.go]: Would you really panic() in a test case? Maybe just asserting that err is nil would be sufficient.

**Comment by peteclark-ft** [comment]
👍

**Comment by sjwells** [comment]
👍

**Comment by khatton-ft** [comment]
TestShouldReturn200AndTrasformedAuthor fails on Windows. (Looks like it's a CRLF/LF problem)

File: author_handlers_test.go (hunk 1 above)
Comment (izzyblues) [inline author_handlers_test.go]: It sounds a good reason to return an error to me.

**Comment by khatton-ft** [comment]
:+1:

**Pull request merged**